Makefile

```diff
@@ -16,10 +16,10 @@ lint:
 	pre-commit run -a --hook-stage manual $(hook)
 
 test:
-	pytest tests --cov-config pyproject.toml --numprocesses 4 --dist loadfile
+	pytest tests --cov-config pyproject.toml
 
 test-no-spark:
-	pytest tests --no-cov --ignore tests/extras/datasets/spark --numprocesses 4 --dist loadfile
+	pytest tests --no-cov --ignore tests/extras/datasets/spark
```

Comment:
These were quite handy in making our tests run slightly faster by parallelising. What's the reason for removing it?

Reply:
See point 1: https://github.com/kedro-org/kedro/pull/1275#issue-1146834454, I could implement Antony's suggestion: https://github.com/kedro-org/kedro/pull/1275#pullrequestreview-891371768